doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -534,20 +534,19 @@ Displaying information about notations
    Prints the current reserved :ref:`keywords <keywords>` and parser tokens, one
    per line. Keywords cannot be used as identifiers.
 
-.. cmd:: Print Grammar @ident
+.. cmd:: Print Grammar {* @ident }
 
-   Shows the grammar for the nonterminal :token:`ident`, which must be one of the following:
+   When no :token:`ident` is provided, shows the whole grammar.
+   Otherwise shows the grammar for the nonterminal :token:`ident`\s. The
+   following are treated specially and print some related grammars:
```

Comment:
```suggestion
   Otherwise shows the grammar for the nonterminal :token:`ident`\s, except for
   the following, which will include some related nonterminals:
```